docs/eventing/samples/writing-event-source-easy-way/Dockerfile

```diff
@@ -4,8 +4,7 @@ FROM node:10
 WORKDIR /usr/src/app
 
 # Install app dependencies
-# A wildcard is used to ensure both package.json AND package-lock.json are copied
-# where available (npm@5+)
+# A wildcard is used to ensure both package.json AND package-lock.json
```

Comment:
Should this still have the "are copied where available", or maybe some other wording - "are copied' without adding where available.

Reply:
I think it's highly likely that developers are using npm@5 or newer, since npm@5 was released with Node * in 2017.